sdk/python/kfp_tekton/tekton.py

```diff
@@ -25,8 +24,10 @@
 DEFAULT_CONDITION_OUTPUT_KEYWORD = "outcome"
 TEKTON_CUSTOM_TASK_IMAGES = [CEL_EVAL_IMAGE]
 
```

Comment:
Need an extra newline here to pass the `make lint`

Reply:
Yeah, not ready to be merged yet, will update.